isb_lib/opencontext_adapter.py

```diff
@@ -113,21 +115,23 @@ def records_in_page(self):
                         self._date_start
                     )
                     data = {}
+                    self._past_date_start = True
                     break
                 # print(json.dumps(record, indent=2))
                 # raise NotImplementedError
                 yield record
                 num_records += 1
-            self.url = data.get("next-json")
```

Comment:
This was the bug here -- previously we reached the end of the records because we were past the start date, and we cleared the current data, but then asked for the next url from the empty data, which set it to null.